CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: Removed path to sigar lib in ``crate.bat`` for Windows.
```

Comment:
this is too detailed, pls change to something like "Fixed node startup on Windows if sigar plugin is deleted"